### PR TITLE
Feature: Adding linkback functionality Author avatar & Name

### PR DESCRIPTION
**Pull request by shredtechular**
This pull request is an attempt at addressing #1331.  This is the simplest/cleanest way that I can think to add this functionality.  If this is too messy, I completely understand and we can close this and I can simply do it locally, but again, I always like to keep customization as minimal as possible to more easily take advantage of future updates.

At any rate, I think this solution does solve the concern noted in #1331 around easily turning it on/off and allowing it for one, some, or many authors depending on the site's structure.

Basically, the code will check for an `author.linkback` yml key and if it is set, then it will add the link around a given author's avatar and name.  If it is not set, then the link will not be generated.

If this is something you'd consider adding in, I can certainly update the docs where applicable, but wanted to see if you'd even consider this before doing that?

### CODE REVIEW

File: _includes/author-profile.html [hunk 1]
```diff
@@ -9,15 +9,34 @@
   {% if author.avatar %}
     <div class="author__avatar">
       {% if author.avatar contains "://" %}
-        <img src="{{ author.avatar }}" alt="{{ author.name }}" itemprop="image">
+        {% assign author_src = author.avatar %}
+        {% assign author_class = "" %}
       {% else %}
-        <img src="{{ author.avatar | absolute_url }}" class="author__avatar" alt="{{ author.name }}" itemprop="image">
+        {% assign author_src = author.avatar | absolute_url %}
+        {% assign author_class = "author__avatar" %}
```
Comment (mmistakes) [inline _includes/author-profile.html]: I think you can safely remove the `author_class` stuff. Looking back at my original code `class="author__avatar"` on the `` element was left in mistakenly. It's applied on the wrapper `div` and not needed.

**Comment by mmistakes** [comment]
This looks pretty clean to me so I'm good with merging in with some minor adjustments.

Left a note above about the `author__avatar` class that was inconsistently applied to the bio image. I think I left that in by mistake, but it's not needed anymore.

The only thing giving me pause is the naming of `linkback`. Not sure if there is something better that is more clear or semantic. So if anyone has other suggestions I'm all ears. If nothing better comes in then `link_back` would follow the style of other `site` variables.

And yes, feel free to add this to the [Configuration docs](https://mmistakes.github.io/minimal-mistakes/docs/configuration/#site-author) and a line in `_config.yml` similar to the others.

**Comment by shredtechular** [comment]
Thanks for the feedback!

I didn't realize that about `author__avatar` so I removed that in my update 890cf66.  I also updated the Configuration docs adding a line for the author site variable.

As far as the name of what variable to use there, I'm also open to suggestions on what might be the most clear.  Instead of `link_back` I changed it to simply `home` with the reasoning that folks that use this feature would probably be wanting to link to either the site's "home" page or an author "home" or "index" page.  Do you like that better or does that still not describe it adequately?

**Comment by mmistakes** [comment]
`home` sounds good to me. Thanks for making the change.
